src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -651,6 +651,14 @@ public Builder inheritProcessOutput(boolean inheritProcessOutput) {
     }
 
     /**
+     * Sets the desired Java SDK path, used in devappserver runs and App Engine standard staging.
+     */
+    public Builder javaSdkPath(Path javaSdkPath) {
```

Comment:
@briandealwis could you file an issue for this?  I think your suggestion makes sense and we should spend some time thinking it through for both devappservers, but I don't want to block this change on it as we need to get a fix out soon.